admin-auth/src/test/kotlin/com/epam/drill/admin/auth/UserRepositoryImplTest.kt

```diff
@@ -209,19 +213,20 @@ class UserRepositoryImplTest {
     }
 
     @Test
-    fun `findById must return even deleted user`() = withTransaction {
-        insertUsers(1..10)
-        val deletedIds = insertUsers(11..15) {
-            it[deleted] = true
+    fun `given existent id, delete must delete user`() = withTransaction {
+        val id = insertUser {
+            it[username] = "foo"
         }
-        insertUsers(16..20)
-        deletedIds.forEach { id ->
 
-            val user = repository.findById(id)
+        repository.deleteById(id)
 
-            assertNotNull(user)
-            assertEquals(id, user.id)
-            assertTrue(user.deleted)
+        assertEquals(0, UserTable.select { UserTable.id eq id }.count())
+    }
+
+    @Test
+    fun `given non-existent id, delete must not fail`() = withTransaction {
+        assertDoesNotThrow {
+            repository.deleteById(404)
```

Comment:
Lets replace `404` with something like `12345`, otherwise its a bit confusing (sine 404 is instantly mentally associatable with a http status)